### PR TITLE
MMT-3062: Fix tests

### PR DESCRIPTION
**Pull request by htranho**

### CODE REVIEW

**Review by dmistry1 — COMMENTED** [review]
Uncomment the after all and there were some screenshot_and_open_image in the files

File: spec/features/manage_cmr/service_entries/create_service_entry_spec.rb [hunk 1]
```diff
@@ -1,30 +1,36 @@
...
-  end
-
-  before do
     collections_response = Cmr::Response.new(Faraday::Response.new(status: 200, body: JSON.parse(File.read('spec/fixtures/cmr_search.json'))))
     allow_any_instance_of(Cmr::CmrClient).to receive(:get_collections_by_post).and_return(collections_response)
-
     login
   end
 
+  # after :all do
```
Comment (cgokey) [inline spec/features/manage_cmr/service_entries/create_service_entry_spec.rb:15]: these should run to cleanup the group creation.

File: spec/features/manage_cmr/service_entries/create_service_entry_spec.rb [hunk 2]
```diff
@@ -1,30 +1,36 @@
...
+  #   allow_any_instance_of(Cmr::CmrClient).to receive(:get_collections_by_post).and_return(collections_response)
+  #   login
+  # end
+
   context 'when the user does not have the required permissions' do
     context 'when viewing the new service entry form' do
       before do
         VCR.use_cassette('echo_soap/service_management_service/service_entries/new', record: :none) do
           visit new_service_entry_path
+          # screenshot_and_open_image
```
Comment (cgokey) [inline spec/features/manage_cmr/service_entries/create_service_entry_spec.rb:33]: remove

File: spec/features/manage_cmr/service_entries/create_service_entry_spec.rb [hunk 3]
```diff
@@ -39,13 +45,24 @@
...
-    before :all do
-      @group_permissions = add_permissions_to_group(@service_entry_group['group_id'], 'create', 'EXTENDED_SERVICE', 'MMT_2')
+    before do
+      VCR.use_cassette("edl/#{File.basename(__FILE__, ".rb")}_vcr", record: :none) do
+        @token = 'jwt_access_token'
+        allow_any_instance_of(ApplicationController).to receive(:echo_provider_token).and_return(@token)
+        allow_any_instance_of(Cmr::UrsClient).to receive(:get_client_token).and_return('client_token')
+        # cmr_client.delete_permission('ACL1200443647-CMR', @token)
+        @group_permissions = add_permissions_to_group(@service_entry_group['group_id'], 'create', 'EXTENDED_SERVICE', 'MMT_2', @token)
+        puts("Group permission=#{@group_permissions}")
```
Comment (cgokey) [inline spec/features/manage_cmr/service_entries/create_service_entry_spec.rb:55]: remove

File: spec/features/manage_cmr/service_entries/delete_service_entry_spec.rb [hunk 4]
```diff
@@ -1,25 +1,28 @@
...
-    @delete_permissions = add_permissions_to_group(@service_entry_group['group_id'], 'delete', 'EXTENDED_SERVICE', 'MMT_2')
   end
 
-  after :all do
-    remove_group_permissions(@delete_permissions['group_id'])
-    VCR.use_cassette('edl', record: :new_episodes) do
-      delete_group(concept_id: @service_entry_group['group_id'])
-    end
-  end
+  # after :all do
```
Comment (cgokey) [inline spec/features/manage_cmr/service_entries/delete_service_entry_spec.rb:16]: these should run to clean up sit

File: spec/features/manage_cmr/service_entries/listing_service_entries_spec.rb [hunk 5]
```diff
@@ -1,19 +1,21 @@
...
+      @service_entry_group = create_group(name: 'Service_Entries_Group_for_Permissions_LIST_03', members: ['admin'])
     end
   end
 
-  after :all do
-    VCR.use_cassette('edl', record: :new_episodes) do
-      delete_group(concept_id: @service_entry_group['group_id'])
-    end
-  end
+  # after :all do
```
Comment (cgokey) [inline spec/features/manage_cmr/service_entries/listing_service_entries_spec.rb:14]: these should run to clean up sit

File: spec/features/manage_cmr/service_entries/listing_service_entries_spec.rb [hunk 6]
```diff
@@ -129,16 +131,21 @@
...
+        end
         VCR.use_cassette('echo_soap/service_management_service/service_entries/list', record: :none) do
           visit service_entries_path
         end
       end
 
-      after do
-        remove_group_permissions(@create_permissions['concept_id'])
-      end
+      # after do
```
Comment (cgokey) [inline spec/features/manage_cmr/service_entries/listing_service_entries_spec.rb:146]: same

File: spec/features/manage_cmr/service_entries/listing_service_entries_spec.rb [hunk 7]
```diff
@@ -147,16 +154,21 @@
...
+        end
         VCR.use_cassette('echo_soap/service_management_service/service_entries/list', record: :none) do
           visit service_entries_path
         end
       end
 
-      after do
-        remove_group_permissions(@update_permissions['concept_id'])
-      end
+      # after do
```
Comment (cgokey) [inline spec/features/manage_cmr/service_entries/listing_service_entries_spec.rb:169]: same

File: spec/features/manage_cmr/service_entries/listing_service_entries_spec.rb [hunk 8]
```diff
@@ -167,16 +179,21 @@
...
+        end
         VCR.use_cassette('echo_soap/service_management_service/service_entries/list', record: :none) do
           visit service_entries_path
         end
       end
 
-      after do
-        remove_group_permissions(@delete_permissions['concept_id'])
-      end
+      # after do
```
Comment (cgokey) [inline spec/features/manage_cmr/service_entries/listing_service_entries_spec.rb:194]: same

**Comment by cgokey** [comment]
Nice that everything passes.  I have an idea to get those ACL calls working without mocking it.   When you get back, let's talk.

**Review by dmistry1 — APPROVED** [review]

**Review by rosycordova — APPROVED** [review]

**Review by tle1989 — APPROVED** [review]

**Pull request merged**